extensions/3DTILES_multiple_contents/0.0.0/README.md

```diff
@@ -19,21 +20,39 @@ Draft
 
 Written against the 3D Tiles 1.0 specification.
 
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/README.md). See [Implicit Tiling](#implicit-tiling)
+
 <!-- omit in toc -->
 ## Contents
 
 - [Overview](#overview)
 - [Concepts](#concepts)
-  - [Metadata](#metadata)
+  - [Metadata Groups](#metadata-groups)
   - [Implicit Tiling](#implicit-tiling)
-- [Schema Updates](#schema-updates)
+    - [Metadata Groups in Implicit Tiling](#metadata-groups-in-implicit-tiling)
+- [Extension Schema Reference](#extension-schema-reference)
+- [`3DTILES_implicit_tiling` Subtree JSON Reference](#3dtiles_implicit_tiling-subtree-json-reference)
 
 
 ## Overview
 
 This extension adds support for multiple contents per tile.
 
-This is useful for datasets that have multiple content layers. Normally layering is achieved by combining contents into a [Composite](../../../specification/TileFormats/Composite/README.md) content, or by placing contents into sibling tiles, or by creating separate tilesets. With this extension content layers can exist cleanly in the same tileset, while allowing contents to be requested independently from each other.
+Contents can be organized into **groups**. For example, each tile could store two different representations of the same data using two contents: a point cloud and a triangle mesh, each representing the same terrain:
```

Comment:
> Contents can be organized into **groups**

The term "group" is specific to `3DTILES_metadata` so I'm not sure if it's right to redefine here. Technically there isn't any association between contents in different tiles until `3DTILES_metadata` is used.

 So I would combine in into the sentence below.

> When this extension is combined with [`3DTILES_metadata`](../../3DTILES_metadata/README.md), arbitrary contents can be organized into **groups**. Each group can also have metadata associated with it.

These first two sentences in the overview should maybe focus on the advantages over 3D Tiles 1.0. Borrowing from the previous iteration, there are three things a tileset author would have to do today:

* Combining contents into a Composite content - downside: not all applications need all the contents contained within
* Creating sibling tiles - awkward, bloats tileset.json
* Using two separate tilesets - inconvenient for asset management, wasteful if the tilesets are using the same tiling scheme (two traversals instead of one)

The point cloud/triangle mesh terrain example can be used to illustrate those points and why this new extension provides a good alternative.